src/libraries/Directory.Build.targets

```diff
@@ -272,4 +273,20 @@
       </ItemGroup>
     </When>
   </Choose>
+
+  <Target Name="LibIntellisenseDocs">
+   <ItemGroup>
+      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)/1033/$(TargetName).xml" Condition="Exists('$(XmlDocFileRoot)/1033/$(TargetName).xml')">
+        <PackagePath>lib/$(TargetFramework)</PackagePath>
+      </TfmSpecificPackageFile>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="RemoveXmlFilesFromBuildOutput" BeforeTargets="GenerateNuspec">
+    <PropertyGroup>
+      <AllowedOutputExtensionsInPackageBuildOutputFolder>.dll;.exe;.winmd;.json;.pri;</AllowedOutputExtensionsInPackageBuildOutputFolder> 
```

Comment:
You want to add a comment with a TODO to remove this after https://github.com/NuGet/NuGet.Client/pull/3980 is in.